examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -163,6 +163,10 @@ EmberAfStatus OTAProviderExample::HandleQueryImage(chip::app::CommandHandler * c
             otaFilePath              = candidate.otaURL;
         }
     }
+    if (mSoftwareVersion.HasValue())
+    {
+        newSoftwareVersion = mSoftwareVersion.Value();
```

Comment:
Maybe this should be moved to the if block under line 148. We probably shouldn't override the software version if using the OTA candidates list (line 156-165)